ci/link-config.json

```diff
@@ -1,12 +1,13 @@
 {
-    "timeout": "20s",
-    "retryOn429": true,
-    "retryCount": 3,
+    "aliveStatusCodes": [200,206,402],
```

Comment:
Not my favorite, but this works around an issue with linking to a payment required wired article.

Reply:
This ONLY changes the aliveStatusCodes when verifying the validity of a URL and does not bypass a paywall.

Reply:
Correct, this workaround is a CI configuration and does not bypass any controls that wired has in place for enforcing payment